othello/apps/tournaments/utils.py

```diff
@@ -8,10 +8,10 @@ def chunks(v, n):
 
 def make_pairings(players, bye_player):
     matches = []
-    players = sorted(players, key=lambda x: -x.ranking)
+    players = players.order_by("-ranking")
 
     for i in range(0, len(players), 2):
-        if i + 1 > len(players):
+        if i + 1 > players.count():
             players.append(bye_player)
```

Comment:
Now that `players` is a `QuerySet`, this will fail. I would convert `players` to a list since you're going to do all this processing on it.

Note that it's still a good idea to do the sorting database-side, since the DB may be more optimized than Python.